task-manager-client/src/main/java/ru/titov/taskmanagerclient/api/controller/Bootstrap.java

```diff
@@ -0,0 +1,23 @@
+package ru.titov.taskmanagerclient.api.controller;
+
+import ru.titov.taskmanagerclient.command.AbstractCommand;
+import ru.titov.taskmanagerserver.endpoint.project.ProjectEndpoint;
+import ru.titov.taskmanagerserver.endpoint.task.TaskEndpoint;
+import ru.titov.taskmanagerserver.endpoint.user.UserEndpoint;
+
+import java.util.Map;
+import java.util.Scanner;
+
+public interface Bootstrap {
+
+    void start();
+
+    Map<String, AbstractCommand> getCommandsMapping();
```

Comment:
bootstrap is responsible for starting, stopping, suspending, resuming project but not managing commands. You need separate class for managing them